artemis/modules/SubdomainEnumeration.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
```

Comment:
the module is not named according to Python module naming convention. for modules it's module_name